doc/configuration.rst

```diff
@@ -481,18 +481,21 @@ We recommend that you copy this set of exit codes to your ``luigi.cfg`` file:
   missing_data=20
   task_failed=30
   scheduling_error=35
+  unknown_reason=38
```

Comment:
... and (2) I suggest we say `38 or 0`. And then explain more detailed when you might want what number.